tonatona-db-postgresql/src/Tonatona/Db/Postgresql.hs

```diff
@@ -81,9 +81,19 @@ data Config = Config
 
 instance FromEnv Config where
   fromEnv =
-    Config
-      <$> envMaybe "TONA_DB_POSTGRESQL_CONN_STRING" .!= "postgresql://myuser:mypass@localhost:5432/mydb"
-      <*> envMaybe "TONA_DB_POSTGRESQL_CONN_NUM" .!= 10
+    let connStr =
+          envDef
+            ( envVar "TONA_DB_POSTGRESQL_CONN_STRING" .||
+              argLong "postgresql-conn-string"
+            )
+            "postgresql://myuser:mypass@localhost:5432/mydb"
+        connNum =
+          envDef
+            ( envVar "TONA_DB_POSTGRESQL_CONN_NUM" .||
+              argLong "postgresql-conn-num"
+            )
```

Comment:
I think it is better to give priority to command line arguments when both exist.
The reason is because environment variables have more wide scope than command line arguments:
* env var is for environment, so some other applications would use the value
* command line arguments are only for the application, so it should be used to overwrite env var configurations